### ISSUE TITLE
The string "rem" within html triggers strange highlighting

### ISSUE THREAD

**Issue by gtelljohann** [isagalaev/highlight.js#1115]
This string:
```
"<h1>
    Lorem Ipsum Times <small>Filler text for fulfilling people</small>
</h1>"
```
when passed into `hljs.highlightAuto` produces:
```
"&lt;h1&gt;
    Lo<span class="hljs-comment">rem Ipsum Times &lt;small&gt;Filler text for fulfilling people&lt;/small&gt;</span>
&lt;/h1&gt;"
```

And that span makes it look very strange

**Comment by isagalaev** [comment]
It should work fine with the latest master code.

**Comment by gtelljohann** [comment]
oh oops, we were using higlightjs, which is updated on github to 9.2.0, but not on npm

**Comment by gtelljohann** [comment]
thanks!

**Comment by isagalaev** [comment]
To be exact, the last released version on npm is 9.2.0, but the fix is in the master branch which hasn't yet been released. The next release is scheduled on April 5th.
